app/src/components/questions/QuestionAnswerCard.vue

```diff
@@ -51,7 +59,21 @@
         },
         methods: {
             ...mapActions('question', ['upVoteAnswer', 'downVoteAnswer']),
+            editAreaToggle() {
+                this.editArea = !this.editArea
+            },
+            updateAnswerText() {
+                this.$emit('updateText', {new: this.newText, id: this.answer.id})
+                this.editAreaToggle()
+            }
         },
+        data() {
+            return{
+                currentUser: localStorage.getItem('username'),
```

Comment:
Use store value instead of localstorage. 
Under computed use             ...mapState('auth', ['isLoggedIn', 'username']),
then you can access it with this.username.

Localstorage is much easier to manipulate than store.